### ISSUE TITLE
Support for KDE neon

### ISSUE THREAD

**Issue by ecgan** [retrohacker/getos#94]
I'm using [KDE neon](https://neon.kde.org) which is based on Ubuntu. When I use getos, it returns the following without `codename` and `release` fields. 

```
Your OS is:{"os":"linux","dist":"Ubuntu Linux"}
```

This is the content of the `lsb-release` file: 

```
DISTRIB_ID=neon
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="KDE neon User Edition 5.16"
```

And just wondering if it matters, this is the content of the `os-release` file:

```
NAME="KDE neon"
VERSION="5.16"
ID=neon
ID_LIKE="ubuntu debian"
PRETTY_NAME="KDE neon User Edition 5.16"
VARIANT="User Edition"
VERSION_ID="18.04"
HOME_URL="http://neon.kde.org/"
SUPPORT_URL="http://neon.kde.org/"
BUG_REPORT_URL="http://bugs.kde.org/"
LOGO=start-here-kde-neon
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

FWIW, I found out this issue when I was trying to get [mongodb-memory-server](https://github.com/nodkz/mongodb-memory-server) running on my machine - it couldn't get the release version number of my KDE neon OS, and assume and download the wrong mongo binary file.

**Comment by ecgan** [comment]
FYI, I went ahead and created PR #95 to support KDE neon. Let me know if it's good.

**Comment by ecgan** [comment]
Any feedback on this please?

**Comment by retrohacker** [comment]
Thank you for this @ecgan! No feedback :smile: this was a great and well thought out PR. It shipped today in 3.1.4!

**Comment by ecgan** [comment]
@retrohacker , thanks for the comment and merging the PR. I thought it would never see the daylight lol.
